dashboard/app/models/school.rb

```diff
@@ -345,8 +396,8 @@ def load_state_cs_offerings(offerings_to_load, is_dry_run)
   # @param filename [String] The CSV file name.
   # @param options [Hash] The CSV file parsing options.
   # @param update_existing [Boolean] Specify whether existing rows should be updated.  Default to true for backwards compatible with existing logic that calls this method to UPSERT schools.
-
-  def self.merge_from_csv(filename, options = CSV_IMPORT_OPTIONS, update_existing = true, is_dry_run: false, new_attributes: [], insert_new: true)
+  # @param limit [Integer] Limits the number of rows parsed from the csv file (for testing). Default to nil for no limit.
```

Comment:
I think I added these arguments without proper comments. Could you do me a solid and add them while you're in here?

```
# @param is_dry_run [Boolean] Allows testing of importing a CSV by rolling back any changes
# @param new_attributes [Array] List of attributes that are being imported for the first time. Allows us to determine which schools have changes to existing data, and which are just adding new attributes
# @param insert_new [Boolean] Determines whether to insert (or if false, skip) importing new schools in this import